src/main/java/WayofTime/bloodmagic/compat/jei/BloodMagicPlugin.java

```diff
@@ -99,4 +99,9 @@ public void register(@Nonnull IModRegistry registry)
         jeiHelper.getNbtIgnoreList().ignoreNbtTagNames(Constants.NBT.DIMENSION_ID);
         jeiHelper.getNbtIgnoreList().ignoreNbtTagNames(Constants.NBT.ITEM_INVENTORY);
     }
+
+    @Override
+    public void onRuntimeAvailable(IJeiRuntime runtime)
```

Comment:
Why implement this? `BlankModPlugin` should be taking care of any unneeded methods.